lndclient/router_client.go

```diff
@@ -102,6 +104,9 @@ type SendPaymentRequest struct {
 	// MaxParts is the maximum number of partial payments that may be used
 	// to complete the full amount.
 	MaxParts uint32
+
+	// KeySend is set to true if the payment request will encode the preimage.
```

Comment:
`payment request` generally refers to the hex-encoded invoice. You probably mean the tlv payload?

Reply:
ptal